node/src/components/gossiper/tests.rs

```diff
@@ -125,7 +125,7 @@ impl reactor::Reactor<TestRng> for Reactor {
         let network = NetworkController::create_node(event_queue, rng);
 
         let (storage_config, _storage_tempdir) = storage::Config::default_for_tests();
-        let storage = Storage::new(&storage_config).unwrap();
+        let storage = Storage::new(WithDir::new(_storage_tempdir.path(), storage_config)).unwrap();
```

Comment:
Same point here about the leading underscore.

Reply:
Same point